src/run.sh

```diff
@@ -24,6 +24,18 @@ getEnv() {
     redis_host=${REDIS_HOST:-redis}
     redis_port=${REDIS_PORT:-6379}
     ipv6=${ENABLE_IPV6:-false}
+    elabftw_user=${ELABFTW_USER:-nginx}
+    elabftw_group=${ELABFTW_GROUP:-nginx}
+    elabftw_userid=${ELABFTW_USERID:-100}
+    elabftw_groupid=${ELABFTW_GROUPID:-101}
+}
+
+# Create user if not exit
+createUser() {
+    if [ "${elabftw_user}" != "nginx" ]; then
+        addgroup -g ${elabftw_groupid} ${elabftw_group}
+        adduser -S -u ${elabftw_userid} -G s-elabftw-gp ${elabftw_user}
```

Comment:
`s-elabftw-gp` should be `${elabftw_group}` here.

Reply:
Oups, fixed too :)